app/js/views/services.jsx

```diff
@@ -69,7 +69,3 @@ export default class Services extends BaseView {
     );
   }
 }
-
-Services.propTypes = {
```

Comment:
Here and below proptypes are the same as in the base class so no need to keep them.

Also plugin that removes propTypes has a small bug/missing point - it can't remove proptypes from the child classes (so propTypes for classes that inherit from BaseView and have their own propTypes won't be removed). We have only 3 of them now so it's not really an issue, but still. If I have time I'll investigate and maybe file issue.